cmd/kubectl-cnpg/main.go

```diff
@@ -79,28 +81,36 @@ func main() {
 
 	logFlags.AddFlags(rootCmd.PersistentFlags())
 	configFlags.AddFlags(rootCmd.PersistentFlags())
+	//	plugin.AddColorControlFlag(hibernateStatusCmd)
+
+	subcommands := []*cobra.Command{
+		backup.NewCmd(),
+		certificate.NewCmd(),
+		destroy.NewCmd(),
+		fence.NewCmd(),
+		fio.NewCmd(),
+		hibernate.NewCmd(),
+		install.NewCmd(),
+		logs.NewCmd(),
+		maintenance.NewCmd(),
+		pgadmin.NewCmd(),
+		pgbench.NewCmd(),
+		promote.NewCmd(),
+		psql.NewCmd(),
+		publication.NewCmd(),
+		reload.NewCmd(),
+		report.NewCmd(),
+		restart.NewCmd(),
+		snapshot.NewCmd(),
+		status.NewCmd(),
+		subscription.NewCmd(),
+		versions.NewCmd(),
+	}
 
-	rootCmd.AddCommand(certificate.NewCmd())
-	rootCmd.AddCommand(destroy.NewCmd())
-	rootCmd.AddCommand(fence.NewCmd())
-	rootCmd.AddCommand(fio.NewCmd())
-	rootCmd.AddCommand(hibernate.NewCmd())
-	rootCmd.AddCommand(install.NewCmd())
-	rootCmd.AddCommand(maintenance.NewCmd())
-	rootCmd.AddCommand(pgbench.NewCmd())
-	rootCmd.AddCommand(promote.NewCmd())
-	rootCmd.AddCommand(reload.NewCmd())
-	rootCmd.AddCommand(report.NewCmd())
-	rootCmd.AddCommand(restart.NewCmd())
-	rootCmd.AddCommand(status.NewCmd())
-	rootCmd.AddCommand(versions.NewCmd())
-	rootCmd.AddCommand(backup.NewCmd())
-	rootCmd.AddCommand(psql.NewCmd())
-	rootCmd.AddCommand(snapshot.NewCmd())
-	rootCmd.AddCommand(logs.NewCmd())
-	rootCmd.AddCommand(pgadmin.NewCmd())
-	rootCmd.AddCommand(publication.NewCmd())
-	rootCmd.AddCommand(subscription.NewCmd())
+	for _, cmd := range subcommands {
+		plugin.AddColorControlFlag(cmd)
```

Comment:
This should be a persisten flag just like --namespace and everything so it will rule any color on any command